newIDE/app/src/Utils/UseDisplayNewFeature.js

```diff
@@ -20,19 +29,23 @@ const useDisplayNewFeature = () => {
 
   const shouldDisplayNewFeatureHighlighting = React.useCallback(
     ({ featureId }: { featureId: Feature }): boolean => {
+      const programOpeningCount = getProgramOpeningCount();
       const settings = featuresDisplaySettings[featureId];
       if (!settings) return false;
 
       const acknowledgments = newFeaturesAcknowledgements[featureId];
       if (!acknowledgments) return true;
```

Comment:
If the user has never acknowledged this feature, won't this "return true" trigger before it even checks the minimumProgramOpeningCount?